camille/source/windiris.py

```diff
@@ -47,33 +51,87 @@ def _sqlite(start_date,
         'Roll': 'roll',
     }, inplace=True)
     df.index.name = 'time'
+    df.index = df.index.tz_convert(tzinfo)
     df.pitch = df.pitch.apply(radians)
     df.roll = df.roll.apply(radians)
 
     return df
 
 
-def windiris(root):
+def windiris(root, tzinfo=pytz.utc):
+    """
+    Creates a function that can be used to read windiris data from
+    specified root directory
+
+    Parameters
+    ----------
+    root : str
+        Path to windiris data root directory
+    tzinfo : datetime.tzinfo
+        Timezone format for returned timeseries
+    Returns
+    -------
+    function (str, datetime.datetime, datetime.datetime, str, str, str)
+        Function for reading windiris time series from db file
+
+        installation : str
+            Installation directory inside root, part of the file name
+            and sql table name at the same time
+        start : datetime.datetime
+            The start time of the data to be read. Must be timezone aware
+        end : datetime.datetime
+            The end time of the data to be read. Must be timezone aware
+        los_id : int or str, optional
+            LOS Index filter. Default None
+        distance : number or str, optional
+            Distance filter. Default None
+        status : int or str, optional
+            RWS Status filter. Default None
+
+        Returns
+        -------
+        pandas.DataFrame
+           Result dataframe
+
+    Examples
+    --------
+
+    Read year data from table 'wind' located under <root>/wind/wind_rtd.db
```

Comment:
Same as above. The user is reading data from an installation for a given time interval. How this data is stored is irrelevant.